Web_Music/Models/PlaylistCreateRequestModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Web_Music.Models
+{
+    public class PlaylistCreateRequestModel
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public int UserId { get; set; }
+        public UserCreateRequestModel UserCreateRequestModel { get; set; }
```

Comment:
No need of UserCreateRequestModel, UserId is enough

Fix the same in other request models